jedi_language_server/server.py

```diff
@@ -122,28 +122,28 @@ def __init__(self, *args, **kwargs):
 
 @SERVER.feature(COMPLETION_ITEM_RESOLVE)
 def completion_item_resolve(
-    server: JediLanguageServer, params: CompletionItem
+    server: JediLanguageServer, params: Any
 ) -> CompletionItem:
     """Resolves documentation and detail of given completion item."""
     markup_kind = _choose_markup(server)
     # note: params is not a CompletionItem
     # but a namedtuple complying with CompletionItem protocol
     item = CompletionItem(
         label=params.label,
-        kind=params.kind,
-        detail=params.detail,
-        documentation=params.documentation,
-        deprecated=params.deprecated,
-        preselect=params.preselect,
-        sort_text=params.sortText,
-        filter_text=params.filterText,
-        insert_text=params.insertText,
-        insert_text_format=params.insertTextFormat,
-        text_edit=params.textEdit,
-        additional_text_edits=params.additionalTextEdits,
-        commit_characters=params.commitCharacters,
-        command=params.command,
-        data=params.data,
+        kind=getattr(params, "kind", None),
+        detail=getattr(params, "details", None),
```

Comment:
This looks wrong, should be `detail` not `details`

Reply:
Huh, I wonder how that slipped in - thanks!